app/api/schema.graphql

```diff
@@ -0,0 +1,20 @@
+type Book {
+  title: String!
+  desceription: String!
+  publishedAt: String!
+  author: Author!
+}
+
+type Author {
+  name: String!
+  biography: String!
+  books: [Book!]!
+  dateOfBirth: String!
```

Comment:
we use `dateOfBirth` and `date_of_birth` can we use consistent casing across the app